server/index.js

```diff
@@ -16,19 +16,131 @@ app.use(bodyParser.urlencoded({ extended: true }));
 app.get('/:productId', (req, res) => {
   res.sendFile(path.join(__dirname, '../client/index.html'));
 });
-
+// Val
 app.get('/itemDetails/:productId', (req, res) => {
-  axios.get(`http://localhost:5000${req.path}`)
+  axios.get(`http://ec2-3-133-108-106.us-east-2.compute.amazonaws.com${req.path}`)
+    .then(({ headers, data }) => {
+      // handle success
+      res.setHeader('content-type', headers['content-type']);
+      res.send(data);
+    })
+    .catch(err => {
+      res.status(500).send(`Error for Val: ${err}`);
+    });
+})
+
+app.get('/info/:productId', (req, res) => {
+  axios.get(`http://ec2-3-133-108-106.us-east-2.compute.amazonaws.com${req.path}`)
     .then(({ headers, data }) => {
       // handle success
       res.setHeader('content-type', headers['content-type']);
       res.send(data);
     })
     .catch(err => {
-      console.error(err);
       res.status(500).send(`Error: ${err}`);
     });
 })
+// Zack
+// app.get('/pictures', (req, res) => {
+//   axios.get(`http://13.56.229.226${req.path}`)
+//     .then((data) => {
+//       res.send(data);
+//     })
+//     .catch((err) => {
+//       console.error(err);
+//       res.status(500).send(`Error: ${err}`);
+//     })
+// })
+
```

Comment:
You should use req.url instead of req.path. Req.url includes the whole query string and Zack's API is using the query params to set the id (`pictures/batch?id=1`).  
```js
const url = server + req.url;
  axios.get(url)
    .then((response) => {
      res.send(response.data);
    })
```
Also make sure you send response.data back.